src/components/CardRegistration/lib.ts

```diff
@@ -0,0 +1,8 @@
+export function getError(error: string) {
+  switch (true) {
+    case error.includes("email"):
+      return "errorEmail";
+    default:
+      return "errorDefault";
+  }
+}
```

Comment:
Não faria sentido o erro vir do back-end? Poderia ate mesmo mandar o erro como o codigo do erro na i18n, fazendo com que não precise ser feita concatenação como foi feita no arquivo CardRegistration.vue, seria so this.$t(variavelComErro)

Reply:
Resolvido